cloud/vsphere/machineactuator.go

```diff
@@ -58,6 +58,7 @@ const (
 	TfConfigFilename       = "terraform.tf"
 	TfVarsFilename         = "variables.tfvars"
 	TfStateFilename        = "terraform.tfstate"
+	startupScriptFilename  = "machine-startup.sh"
```

Comment:
Follow the pattern of TitleCase please.

Reply:
TitleCase is used to make it public, what is the reason to make this const public?